### ISSUE TITLE
Style/MultilineMethodCallBraceLayout flagging false positive?

### ISSUE THREAD

**Issue by NekoNova** [bbatsov/rubocop#3229]
When running this cop with the configuration:

```
Style/MultilineMethodCallBraceLayout:
  EnforcedStyle: new_line
```

It flags single line calls as wrong, forcing the bracket to be on the new line.

I have the following spec for example:

```
# frozen_string_literal: true
require 'rails_helper'

describe ::MyUploader do
  subject { ::MyUploader.new }

  let(:cache_dir) { Rails.root.join('tmp/uploads') }

  describe '#cache_dir' do
    it 'it has the correct dir for the cache store' do
      expect(subject.cache_dir).to eq cache_dir
    end
  end
end
```

This one is flagged:

```
spec/uploaders/my_uploader_spec.rb:11:7: C: Closing method call brace must be on the line after the last argument.
      expect(subject.cache_dir).to eq cache_dir
      ^^^^^^^^^^^^^^^^^^^^^^^^^
```
--------

## Expected behavior
This cop should only flag incorrect multi-line method calls as its name implies.

## Actual behavior
Flags every method call.

## Steps to reproduce the problem
enable the cop with given configuration

This is extremely important! Providing us with a reliable way to reproduce
a problem will expedite its solution.

## RuboCop version

Include the output of `rubocop -V`: 0.40.0
`(using Parser 2.3.1.2, running on ruby 2.2.5 x86_64-darwin15)`

**Comment by NekoNova** [comment]
## Update
I checked the rules for `Style/MultilineHashBraceLayout` etc.
Basically the new multiline cops added in version 40 all suffer from the same behaviour.

```
spec/support/user_filter_support.rb:17:33: C: Closing hash brace must be on the line after the last hash element.
      let(:default_form_hash) { { responsible_user_ids: admin_user.id } }
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
The above example is a single line getting flagged as well.

**Comment by Aqualon** [comment]
Fixed by https://github.com/bbatsov/rubocop/issues/3133?

**Comment by jonas054** [comment]
Yes. Well actually, #3133 is an issue, but it's fixed by [this commit](https://github.com/bbatsov/rubocop/commit/b8f8782dcc89aac44707b68c6d6fa1bd3f472329) included in the PR #3182.
